.github/workflows/workflow.yml

```diff
@@ -13,12 +13,12 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest, macOS-latest, windows-latest]
-        node-version: [14.0.0, '*']
+        node-version: [18.19.0, '*']
```

Comment:
As we still need to support Node 14 in our build shouldn't we keep it that way here as well?

Reply:
ESM support was still experimental in Node 14: https://nodejs.org/docs/v14.2.0/api/esm.html#esm_enabling

That's why the tests failed before I updated this here: https://github.com/netlify/functions/actions/runs/7113273604/job/19364886077

They became stable in Node 16, so maybe we can set the minimum version to Node 16?

Netlify Build doesn't have a runtime dependency on @netlify/functions, so it should be fine to update this independently. If we're intending to raise the minimum Build version anyway, we might as well start here! Especially since users can choose to stay on the old version of this package, until they upgraded their Node.js to v16+.